x-pack/plugins/session_view/public/components/ProcessTreeNode/index.tsx

```diff
@@ -93,12 +110,12 @@ export function ProcessTreeNode({
   };
 
   const renderSessionLeader = () => {
-    const { name, user } = process.getLatest().process;
+    const { name, executable, user } = process.getDetails().process;
```

Comment:
Hmmm just thinking about this again...but Im wondering if we should declare these as computed values so we can avoid any edge cases that might come up with these not being called?

```
const processDetails = useMemo(() => {
  return process.getDetails();
}, [process]);

const hasExec = useMemo(() => {
  return process.hasExec();
}, [process]);

etc
```

This might seem like an arbitrary change but it clearly tells the viewer that these might be side effects, also tells the viewer clearly when these values should be recomputed?

I don't know just my opinion😄